provider/cmd/pulumi-resource-aws/main.go

```diff
@@ -17,11 +17,16 @@
 package main
 
 import (
+	_ "embed"
```

Comment:
Curious why this is necessary. I'm sure it is - I just don't understand this line since I thought (perhaps incorrectly) that other `go` commands do not require an import.